src/components/MessagesList/MessagesList.vue

```diff
@@ -412,14 +412,20 @@ export default {
 		},
 
 		softUpdateByDateGroups(oldDateGroups, newDateGroups) {
-			// Check if we have this group in the old list already and it is unchanged
-			Object.keys(newDateGroups).forEach(dateTimestamp => {
-				if (oldDateGroups[dateTimestamp]) {
-					// the group by date has changed, we update its content (groups by author)
-					this.softUpdateAuthorGroups(oldDateGroups[dateTimestamp], newDateGroups[dateTimestamp], dateTimestamp)
-				} else {
-					// the group is new
-					this.messagesGroupedByDateByAuthor[dateTimestamp] = newDateGroups[dateTimestamp]
+			const dateTimestamps = new Set([...Object.keys(oldDateGroups), ...Object.keys(newDateGroups)])
+
+			dateTimestamps.forEach(dateTimestamp => {
+				if (newDateGroups[dateTimestamp]) {
+					if (oldDateGroups[dateTimestamp]) {
+						// the group by date has changed, we update its content (groups by author)
+						this.softUpdateAuthorGroups(oldDateGroups[dateTimestamp], newDateGroups[dateTimestamp], dateTimestamp)
+					} else {
+						// the group is new
+						this.messagesGroupedByDateByAuthor[dateTimestamp] = newDateGroups[dateTimestamp]
+					}
+				} else if (oldDateGroups[dateTimestamp]) {
```

Comment:
```suggestion
				} else {
```
Not sure we need this check. If we reach this block, timestamp is definitely given by oldDateGroups

Reply:
Yeah, it is not needed at the moment